subprojects/jvm-services/src/main/java/org/gradle/jvm/toolchain/internal/IntellijIdeaInstallationSupplier.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.jvm.toolchain.internal;
+
+import org.gradle.api.provider.Provider;
+import org.gradle.api.provider.ProviderFactory;
+import org.gradle.internal.os.OperatingSystem;
+
+import javax.inject.Inject;
+import java.io.File;
+import java.util.Set;
+
+public class IntellijIdeaInstallationSupplier extends AutoDetectingInstallationSupplier {
```

Comment:
I think we can remove "Idea" from the name, just "IntelliJ" should be enough.

Reply:
Fixed with 0d44e23953a4aba6bc3e6e04e990337a99767778.